app/src/main/java/org/woowatechcamp/githubapplication/presentation/home/notifications/NotificationsFragment.kt

```diff
@@ -99,6 +94,24 @@ class NotificationsFragment : Fragment() {
                 }
             }.launchIn(lifecycleScope)
 
+        viewModel.notiCommentState.flowWithLifecycle(viewLifecycleOwner.lifecycle)
+            .onEach {
+                when (it) {
+                    is UiState.Success -> {
+                        val item = it.value
+                        with(notiAdapter.currentList) {
+                            find { noti -> noti.id == item.id }?.let { found ->
+                                found.commentNum = item.commentNum
+                                notiAdapter.notifyItemChanged(indexOf(found))
+                            }
+                        }
+                    }
+                    is UiState.Error -> {
+                        showSnackBar(binding.root, it.msg, requireActivity()) }
+                    else -> {}
+                }
+            }.launchIn(lifecycleScope)
```

Comment:
이부분도 프래그먼트니까 메모리 세이프하게

```suggestion
            }.launchIn(viewLifecycleOwner.lifecycleScope)
```

이렇게 바꿔주시면 좋을 것 같아요!

Reply:
네 이것도 반영하겠습니다!